ci-amethyst.yml

```diff
@@ -9,17 +9,18 @@ variables:
   travis_cookbooks_branch: "{{ env `TRAVIS_COOKBOOKS_BRANCH` }}"
   travis_cookbooks_edge_branch: master
   travis_cookbooks_sha: "{{ env `TRAVIS_COOKBOOKS_SHA` }}"
+  travis_uid: "{{ env `TRAVIS_UID` }}"
 builders:
 - type: googlecompute
   name: googlecompute
   communicator: ssh
   ssh_timeout: 10m
   ssh_port: 22
-  ssh_username: travis
+  ssh_username: amethyst
```

Comment:
It's a little bitty nitpick, but if we have to have a username other than `ubuntu` and `travis`, then my vote is for `packer` so that we can use the same one across all templates.